schema/query-builder/mixins/name-search.js

```diff
@@ -1,14 +1,22 @@
+const tsquery = require('pg-tsquery');
+const stringify = tsquery();
+
 module.exports = (Base) => {
 
   class NameSearch extends Base {
     whereNameMatch(search, prefix = 'profiles') {
       if (Array.isArray(search)) {
         search = search[0];
       }
-      const parts = search.split(' ').map(p => `${p}:*`).join(' & ');
-      const q = `(to_tsvector(unaccent(${prefix}.first_name)) || to_tsvector(unaccent(${prefix}.last_name))) @@ to_tsquery('${parts}')`;
 
-      return this.whereRaw(q);
+      const query = stringify(search.split(' ').map(p => p + '*').join(' '));
```

Comment:
is `query` a string here?

Reply:
I'm not sure of what you're getting at, but I've renamed the function.

`query` is a string though.

Reply:
It was more the var name - we tend to use `query` for the query object, but that looked like it was a search string, so it was just a bit confusing.